test/python/converters/test_circuit_to_dagdependency_v2.py

```diff
@@ -0,0 +1,71 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test for the converter dag dependency to circuit and circuit to dag
+dependency."""
+
+import unittest
+from test import QiskitTestCase
+
+from qiskit.converters.dagdependency_to_circuit import dagdependency_to_circuit
+from qiskit.converters.circuit_to_dagdependency_v2 import circuit_to_dagdependency_v2
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+
+
+class TestCircuitToDagCanonical(QiskitTestCase):
+    """Test QuantumCircuit to DAGDependency."""
```

Comment:
Should the name of the class be `TestCircuitToDagDependency` (not that it really matters).

Reply:
Ok.